sympy/sets/tests/test_sets.py

```diff
@@ -894,3 +895,19 @@ def test_issue_9808():
     assert Complement(FiniteSet(y), FiniteSet(1)) == Complement(FiniteSet(y), FiniteSet(1), evaluate=False)
     assert Complement(FiniteSet(1, 2, x), FiniteSet(x, y, 2, 3)) == \
         Complement(FiniteSet(1), FiniteSet(y), evaluate=False)
+
+
+def test_big_union():
+    from sympy import IndexedBase
+    i = Symbol('i')
+    X = IndexedBase('X')
+    assert BigUnion(X, i, S.Naturals) == BigUnion(X, i, S.Naturals)
+    assert BigUnion(FiniteSet(Interval(1, 2), FiniteSet(3)), i, FiniteSet(1, 2)) == Union(Interval(1, 2), FiniteSet(3))
+
+
+def test_big_intersection():
+    from sympy import IndexedBase
+    i = Symbol('i')
+    X = IndexedBase('X')
+    assert BigIntersection(X, i, S.Naturals) == BigIntersection(X, i, S.Naturals)
```

Comment:
To me it seems like it is not the best way to test this at all. Any suggestion on this point?
(since it is like `expression` == `expression`)